Unfettered: Hunger Stories (#11)
Hurleveur/endless-sky: data/hai/unfettered side missions.txt

	on enter
		# SPOILER: he doesn't want them to get destroyed by the Hai because that would damage Alpha-Unfettered reputations, also he takes it as a challenge to his strength (except he's on your ship)
		dialog `Upon entering the system you see multiple Cerberus ships, ready to attack you. Choot'k sends a message to the Hai. "Do not interfere, it would damage both my honor and <last>'s!" He did not ask for your opinion, but thankfully you have seen worse than a bunch of pirates.`

mission "Unfettered: Hunger Stories 1"

> 	name `Transport family to <destination>`

Reviewer: ```suggestion
	minor
	name `Transport a family of <passengers> to <destination>`
```

	description `Transport a bereaved Unfettered family to <destination>.`
	source "Firelode"

> 	destination "Hai-home"

Reviewer: ```suggestion
	destination "Hai-home"
	passengers 2
```
you forgot the family! also we usually add a blocked message but here I dont think we need to since its not a main story?

> 	clearance `You contact the authorities when you enter the system and they say they will have a team waiting when you land.`

Reviewer: ```suggestion
	clearance `You contact the authorities and they say they will have a team waiting when you land.`
```
this happens when you ask to land not when you enter the system

			`You are walking past one of the Unfettered government offices in the spaceport when a woman walks out clearly distraught. Her clothes hang loose on her small frame. An infant clings tightly to her fur and a small boy walks behind her with a sombre expression. A collection of silver oak leaves is pinned to his jacket.`
			choice
				`	(Approach her.)`
				`	(Get on with my business.)`
					goto business

			`	Your approach startles her out of her distraction and she pulls the boy close. When you speak to her it's clear she doesn't understand you. You look around for an Unfettered who could translate, but she grabs your sleeve and shakes her head. Eventually one of your crew who understands a little Hai gives it a try.`
			`	"They have told me my husband is dead. The theiving raptors killed him. And now the boy has a medal but no father. They gave me his death money but there is no food and I don't know what we will do."`
			`	You know that the Hai send food aid to the Unfettered and you've not seen anyone obviously hungry in Hai space. They would probably take this family in.`
			choice
				`	(Offer to take them to <destination>.)`
				`	(Offer your condolences and move on.)`

> 					goto "condolences"

Reviewer: ```suggestion
					goto condolences
```
dont need those commas when its in one word; but smth like "big word" would need them

> 			`	While it's awkard to wait for your crew to translate, it gives you time to choose your words. "Your brothers and sisters in the south have food and shelter for your family. If you want to go, I can take you there."`

Reviewer: ```suggestion
			`	While it's awkard to wait for your crew to translate, it gives you time to choose your words. "Your brothers and sisters in the south have food and shelter for your family. I can take you there if you want."`
```

			`	She hesitates but then the baby starts to cry. She looks at you with tears in her eyes and nods. You send some crew with her to help her pack her things and bring them to the ship.`
				accept
			
			label "business"
			`	The Unfettered mother never notices you as you pass by. You see the family later in a long queue for food rations.`
				decline

			label "condolences"

> 			`	You tell her you are sorry for her loss. You see anger flare across her features as you turn to leave and if she wasn't holding an infant she might well have attacked you. "What good are your words monkey-man?" she shouts. " Will they feed my children? Did you come all this way to make fun of us? We have to choose between dying for a new world or starving on this one. Go back through the wormhole and give your words to your own kind!"`

Reviewer: ```suggestion
			`	You tell her you are sorry for her loss. You see anger flare across her features as you turn to leave and if she wasn't holding an infant she might well have attacked you. "What good are your words monkey-man?" she shouts. "Will they feed my children? Did you come all this way to make fun of us? We have to choose between dying for a new world or starving on this one. Go back through the wormhole and give your words to your own kind!"`
```

> 			`	A group of Unfettered roughly screen you off from the distraught mother and tell you to go to leave. She continues screaming at you and crying as you walk away.`

Reviewer: ```suggestion
			`	A group of Unfettered roughly screen you off from the distraught mother and tells you to leave. She continues screaming at you and crying as you walk away.`
```

				decline

	on complete
		payment 300000
		conversation
			`You are suprised to see a military honor guard in addition to what you assume are social services. They greet the family with a keening trill that your crewmember says is a sound of mourning. The Hai move in close and surround the group in what seems to be a group hug and then they move together into the spaceport. One of the Hai soldiers approaches you and says, "Thank you for brining our family back together" and gives you your payment of <payment>.`

> mission "Unfettered: Hunger Stories 2"

Reviewer: I don't mind this one but I'm not sure why it happens only during an hunger situation?

> 	name `Transport warrior to <destination>`

Reviewer: same, minor and the passenger!

	description `Transport an Unfettered warrior to <destination>.`
	source "Darkcloak"
	destination "Makerplace"
	to offer
		has "unfettered: hunger"
	on offer
		conversation
			`"Human!" The cry catches your attention not only because of the volume of the shout, but primarily because it's the most respectful term anyone has used for you in Unfettered space. You turn and see a young Hai warrior striding toward you. At least you think he's young. He's certainly thinner than most of the Hai you see south of Wah-Ki. You notice that his boots seem made more of mending tape than material. He's not wearing the ornately decorated knives that many of the True Hai carry, but he does have sheaths for them. Instead, he carries a long staff that looks to be made of one of the light but rigid Hai alloys.You can see that scars from many fights disturb the pattern of his fur.`

> 			`	"Human," he says again when he reaches you, "you should not promenade yourself in our star market without worthy guidance. This warrior can tell you what trader tongues are saying because I know your backward speech. Fast fingers not grab human things while I watch," he adds, emphasing the point by banging the staff on the floor and making it ring. You notice the flow of traffic begins to part around you as other traders and travellers give him room.`

Reviewer: I like how he speaks bad english on purpose

			`	You see him around the space port a few times: once working as a porter, another loading cargo. Both seem like odd jobs for someone with his apparent skills and neither will pay very well. As you are heading back to your ship you see him at one of the small food stalls that sells really cheap noodle cups and you wonder if that's the only time he's eaten today.`
				decline

			label "laugh"
			`	You find yourself flat on your back looking up at the young Unfettered. He's holding your sidearm and point it at you with the safety off. You later decide that he must have swept your legs with the staff and grabbed your weapon, but it happened so swiftly you never saw it.`
			`	"I was wrong," he says as he secures your weapon inside his jacket. "I though you warrior. Have respect. You just monkey after all."`
			`	He walks off causally, never bothering to turn around.`
				decline

			label "escort"
			`	His eyes narrow when you tell him to leave. "Mistake to refuse generosity of True Hai," he says. You get the distinct impression that he'd take it further if he wasn't on your ship. As it is, he leaves peacefully and you close the hatch.`
				decline

> 			label "transport"

Reviewer: ```suggestion
			label transport
```
you should remove all of these commas for one word labels

				accept

	on complete
		payment 500
		conversation
			`When you arrive at <destination>, the young Hai guides you around the spaceport as you take care of your business. You don't see his staff; so, either it's too provocative to carry in Hai space or it's collapsible and he has it somewhere on his person. He offers you comments on the Hai conversations going on around you where they are either about you or your business. While he is still stays close to you, he seems to natually relax into the flow of Hai culture.`
			`	You lose track of him somewhere near the end of the day and he doesn't show back up at the ship. When you check his bunk, you find the <payment> you paid him on <origin> with a crudely lettered note that says, "FOR TRANSPORT."`

mission "Unfettered: Hunger Stories 3"

> 	name `Encounter a gang of Unfettered children.`
> 	description `Deal with a gang of Unfettered children.`

Reviewer: ```suggestion
	minor
```
the mission is never accepted so no need for name and desc I think

			choice
				`	(Pay them.)`
					goto "bribe"
				`	(Grab my weapon.)`
					goto "fight"

			label "bribe"	
			`	You see movement in your peripheral vision just as you feel the credits pulled from your hand. One of the children off to your side darted forward before you could react and is now handing your credits to the leader. She checks the amound and then gives you a satisfied smile. Aty credits nod, the group vanishes into the life of the street.`
			`	You make your way quickly back to the spaceport, certain that the kids are shadowing you. It's with some relief that you make it through the security checkpoint.`
				decline
			
			label "fight"
			`	You reach for your sidearm only to find that it isn't there. It seems like a dozen pairs of paws are on you, some tearing at your clothes and some climbing on you. You spin around to try and shake them off. You end up rolling on the ground and then, as suddenly as it began, everything stops and you are alone in the street. You check yourself quickly and you seem to be uninjured. Your clothes are in tatters, though, and your sidearm, communicator, and bag are nowhwere to be seen.`

> 			`	You make your way back to the spaceport. For once, it's useful to be the alien. The guards are quickly able to quickly clear you even without documentation. You hear them laughing as you head for the safety of your ship. You've no doubt that the story of he monkey robbed by children is already making the rounds.`

Reviewer: make the player lose actual money? (I did it)

			`	"They have told me my husband is dead. The theiving raptors killed him. And now the boy has a medal but no father. They gave me his death money but there is no food and I don't know what we will do."`
			`	You know that the Hai send food aid to the Unfettered and you've not seen anyone obviously hungry in Hai space. They would probably take this family in.`
			choice
				`	(Offer to take them to <destination>.)`
				`	(Offer your condolences and move on.)`
					goto "condolences"
			`	While it's awkard to wait for your crew to translate, it gives you time to choose your words. "Your brothers and sisters in the south have food and shelter for your family. I can take you there if you want."`
			`	She hesitates but then the baby starts to cry. She looks at you with tears in her eyes and nods. You send some crew with her to help her pack her things and bring them to the ship.`
				accept
			

> 			label "business"

Reviewer: ```suggestion
			label business
```

			`	The Unfettered mother never notices you as you pass by. You see the family later in a long queue for food rations.`
				decline

> 			label "condolences"

Reviewer: ```suggestion
			label condolences
```

			`	You tell her you are sorry for her loss. You see anger flare across her features as you turn to leave and if she wasn't holding an infant she might well have attacked you. "What good are your words monkey-man?" she shouts. "Will they feed my children? Did you come all this way to make fun of us? We have to choose between dying for a new world or starving on this one. Go back through the wormhole and give your words to your own kind!"`
			`	A group of Unfettered roughly screen you off from the distraught mother and tells you to leave. She continues screaming at you and crying as you walk away.`
				decline

	on complete
		payment 300000
		conversation
			`You are suprised to see a military honor guard in addition to what you assume are social services. They greet the family with a keening trill that your crewmember says is a sound of mourning. The Hai move in close and surround the group in what seems to be a group hug and then they move together into the spaceport. One of the Hai soldiers approaches you and says, "Thank you for brining our family back together" and gives you your payment of <payment>.`

mission "Unfettered: Hunger Stories 2"

> 	name `Transport warrior to <destination>`

Reviewer: ```suggestion
	minor
	name `Transport a Hai warrior to <destination>`
```

> 	description `Transport an Unfettered warrior to <destination>.`

Reviewer: I think it could have more meat to it here

	source "Darkcloak"

> 	destination "Makerplace"

Reviewer: ```suggestion
	destination "Makerplace"
	passengers 1
```

			`	"Human," he says again when he reaches you, "you should not promenade yourself in our star market without worthy guidance. This warrior can tell you what trader tongues are saying because I know your backward speech. Fast fingers not grab human things while I watch," he adds, emphasing the point by banging the staff on the floor and making it ring. You notice the flow of traffic begins to part around you as other traders and travellers give him room.`
			`	It takes you a moment to parse what he is saying as his accent is difficult to understand. You would guess he learned Human language in class but has never spoken with native speakers. You think he is offering to be your translator and bodyguard.`
			choice
				`(Hire him.)`

> 					goto "hire"

Reviewer: ```suggestion
					goto hire
```

				`(Politely decline.)`

> 					goto "polite"

Reviewer: ```suggestion
					goto polite
```

				`(Laugh at him and walk away.)`

> 					goto "laugh"

Reviewer: ```suggestion
					goto laugh
```

> 			label "hire"

Reviewer: ```suggestion
			label hire
```

			`	The young Unfettered proves diligent in his work. He steers you away from some vendors and towards others and the prices and quality are at least competitive. Several times his staff has flashed out and blocked people who might have bumped into you. While some of the vendors you traded with were amused by his translations, most worked through him rather than speaking to you directly. He even kept watch while you were eating, though he did seem to be distracted by the food.`
			`	At the end of the day, he follows you to the <ship> and you pay him his fee. He surprises you though, by following you on board. "I will watch over you until you make it back to the Fettered worlds. You should be safe enough on <destination>." From the prices you saw in the resturants and shops, the amount you paid would feed him for a few days at most.`
			choice
				`	(Give him transport.)`

> 					goto "transport"

Reviewer: ```suggestion
					goto transport
```

				`	(Escort him off the ship.)`

> 					goto "escort"

Reviewer: ```suggestion
					goto escort
```

> 			label "polite"

Reviewer: ```suggestion
			label polite
```

			`	You tell him that you can take care of yourself. He nods to acknowledge your decision but maintains eye contact. He then turns and walks away.`
			`	You see him around the space port a few times: once working as a porter, another loading cargo. Both seem like odd jobs for someone with his apparent skills and neither will pay very well. As you are heading back to your ship you see him at one of the small food stalls that sells really cheap noodle cups and you wonder if that's the only time he's eaten today.`
				decline

> 			label "laugh"

Reviewer: ```suggestion
			label laugh
```

			`	You find yourself flat on your back looking up at the young Unfettered. He's holding your sidearm and point it at you with the safety off. You later decide that he must have swept your legs with the staff and grabbed your weapon, but it happened so swiftly you never saw it.`
			`	"I was wrong," he says as he secures your weapon inside his jacket. "I though you warrior. Have respect. You just monkey after all."`
			`	He walks off causally, never bothering to turn around.`
				decline

> 			label "escort"

Reviewer: ```suggestion
			label escort
```

			`	You lose track of him somewhere near the end of the day and he doesn't show back up at the ship. When you check his bunk, you find the <payment> you paid him on <origin> with a crudely lettered note that says, "FOR TRANSPORT."`

mission "Unfettered: Hunger Stories 3"
	minor
	source "Warfeed"
	to offer
		has "unfettered: hunger"
	on offer
		conversation
			`You take some time to explore outside the secure facilities at the spaceport. You can see long lines of people waiting for the food rations that seem to be the main cargo delivered here.`
			`	On your way back to your ship, a gang of Unfettered children block your path and move to encircle you. One of the older girls steps forward and in halting Human says, "Trespass. 500 credits. Then go." You can see half a dozen of them and all of them are half your size.`
			choice
				`	(Pay them.)`

> 					goto "bribe"

Reviewer: ```suggestion
					goto bribe
```

				`	(Grab my weapon.)`

> 					goto "fight"

Reviewer: ```suggestion
					goto fight
```

> 			label "bribe"	

Reviewer: ```suggestion
			label bribe
			action
				payment -500
```

			`	You see movement in your peripheral vision just as you feel the credits pulled from your hand. One of the children off to your side darted forward before you could react and is now handing your credits to the leader. She checks the amound and then gives you a satisfied smile. Aty credits nod, the group vanishes into the life of the street.`
			`	You make your way quickly back to the spaceport, certain that the kids are shadowing you. It's with some relief that you make it through the security checkpoint.`
				decline
			

> 			label "fight"

Reviewer: ```suggestion
			label fight
			action
				payment -5000
```

			choice
				`	(Approach her.)`
				`	(Get on with my business.)`
					goto business

			`	Your approach startles her out of her distraction and she pulls the boy close. When you speak to her it's clear she doesn't understand you. You look around for an Unfettered who could translate, but she grabs your sleeve and shakes her head. Eventually one of your crew who understands a little Hai gives it a try.`
			`	"They have told me my husband is dead. The theiving raptors killed him. And now the boy has a medal but no father. They gave me his death money but there is no food and I don't know what we will do."`
			`	You know that the Hai send food aid to the Unfettered and you've not seen anyone obviously hungry in Hai space. They would probably take this family in.`
			choice
				`	(Offer to take them to <destination>.)`
				`	(Offer your condolences and move on.)`
					goto condolences

> 			`	While it's awkard to wait for your crew to translate, it gives you time to choose your words. "Your brothers and sisters in the south have food and shelter for your family. I can take you there if you want."`

Reviewer: ```suggestion
			`	While it's awkward to wait for your crew to translate, it gives you time to choose your words. "Your brothers and sisters in the south have food and shelter for your family. I can take you there if you want."`
```

			`	She hesitates but then the baby starts to cry. She looks at you with tears in her eyes and nods. You send some crew with her to help her pack her things and bring them to the ship.`
				accept
			
			label business
			`	The Unfettered mother never notices you as you pass by. You see the family later in a long queue for food rations.`
				decline

			label condolences
			`	You tell her you are sorry for her loss. You see anger flare across her features as you turn to leave and if she wasn't holding an infant she might well have attacked you. "What good are your words monkey-man?" she shouts. "Will they feed my children? Did you come all this way to make fun of us? We have to choose between dying for a new world or starving on this one. Go back through the wormhole and give your words to your own kind!"`
			`	A group of Unfettered roughly screen you off from the distraught mother and tells you to leave. She continues screaming at you and crying as you walk away.`
				decline

	on complete
		payment 300000
		conversation

> 			`You are suprised to see a military honor guard in addition to what you assume are social services. They greet the family with a keening trill that your crewmember says is a sound of mourning. The Hai move in close and surround the group in what seems to be a group hug and then they move together into the spaceport. One of the Hai soldiers approaches you and says, "Thank you for brining our family back together" and gives you your payment of <payment>.`

Reviewer: ```suggestion
			`You are surprised to see a military honor guard in addition to what you assume are social services. They greet the family with a keening trill that your crewmember says is a sound of mourning. The Hai move in close and surround the group in what seems to be a group hug and then they move together into the spaceport. One of the Hai soldiers approaches you and says, "Thank you for brining our family back together" and gives you your payment of <payment>.`
```

			`	It takes you a moment to parse what he is saying as his accent is difficult to understand. You would guess he learned Human language in class but has never spoken with native speakers. You think he is offering to be your translator and bodyguard.`
			choice
				`(Hire him.)`
					goto hire
				`(Politely decline.)`
					goto polite
				`(Laugh at him and walk away.)`
					goto laugh
			
			label hire
			`	The young Unfettered proves diligent in his work. He steers you away from some vendors and towards others and the prices and quality are at least competitive. Several times his staff has flashed out and blocked people who might have bumped into you. While some of the vendors you traded with were amused by his translations, most worked through him rather than speaking to you directly. He even kept watch while you were eating, though he did seem to be distracted by the food.`

> 			`	At the end of the day, he follows you to the <ship> and you pay him his fee. He surprises you though, by following you on board. "I will watch over you until you make it back to the Fettered worlds. You should be safe enough on <destination>." From the prices you saw in the resturants and shops, the amount you paid would feed him for a few days at most.`

Reviewer: ```suggestion
			`	At the end of the day, he follows you to the <ship> and you pay him his fee. He surprises you though, by following you on board. "I will watch over you until you make it back to the Fettered worlds. You should be safe enough on <destination>." From the prices you saw in the restaurants and shops, the amount you paid would feed him for a few days at most.`
```

			`	You see him around the space port a few times: once working as a porter, another loading cargo. Both seem like odd jobs for someone with his apparent skills and neither will pay very well. As you are heading back to your ship you see him at one of the small food stalls that sells really cheap noodle cups and you wonder if that's the only time he's eaten today.`
				decline

			label laugh
			`	You find yourself flat on your back looking up at the young Unfettered. He's holding your sidearm and point it at you with the safety off. You later decide that he must have swept your legs with the staff and grabbed your weapon, but it happened so swiftly you never saw it.`
			`	"I was wrong," he says as he secures your weapon inside his jacket. "I though you warrior. Have respect. You just monkey after all."`
			`	He walks off causally, never bothering to turn around.`
				decline

			label escort
			`	His eyes narrow when you tell him to leave. "Mistake to refuse generosity of True Hai," he says. You get the distinct impression that he'd take it further if he wasn't on your ship. As it is, he leaves peacefully and you close the hatch.`
				decline

			label transport

> 			`	You drop the Hai off at the galley and tell him to eat as much as he wants while you prepare the ship for launch. When you return after reaching orbit, his belly is noticably distendend. He seems more relaxed than he did at the spaceport, but that could just be the on-coming food coma. You take him to the uniform locker and find him some boots and a jumpsuit, though you have to make some quick alterations with your knife to accommodate his tail. He asks for a sewing kit and says that he can finish the job.`

Reviewer: ```suggestion
			`	You drop the Hai off at the galley and tell him to eat as much as he wants while you prepare the ship for launch. When you return after reaching orbit, his belly is noticeably distended. He seems more relaxed than he did at the spaceport, but that could just be the on-coming food coma. You take him to the uniform locker and find him some boots and a jumpsuit, though you have to make some quick alterations with your knife to accommodate his tail. He asks for a sewing kit and says that he can finish the job.`
```

			`	You lose track of him somewhere near the end of the day and he doesn't show back up at the ship. When you check his bunk, you find the <payment> you paid him on <origin> with a crudely lettered note that says, "FOR TRANSPORT."`

mission "Unfettered: Hunger Stories 3"
	minor
	source "Warfeed"
	to offer
		has "unfettered: hunger"
	on offer
		conversation
			`You take some time to explore outside the secure facilities at the spaceport. You can see long lines of people waiting for the food rations that seem to be the main cargo delivered here.`
			`	On your way back to your ship, a gang of Unfettered children block your path and move to encircle you. One of the older girls steps forward and in halting Human says, "Trespass. 500 credits. Then go." You can see half a dozen of them and all of them are half your size.`
			choice
				`	(Pay them.)`
					goto bribe
				`	(Grab my weapon.)`
					goto fight

			label bribe
			action
				payment -500

> 			`	You see movement in your peripheral vision just as you feel the credits pulled from your hand. One of the children off to your side darted forward before you could react and is now handing your credits to the leader. She checks the amound and then gives you a satisfied smile. Aty credits nod, the group vanishes into the life of the street.`

Reviewer: ```suggestion
			`	You see movement in your peripheral vision just as you feel the credits pulled from your hand. One of the children off to your side darted forward before you could react and is now handing your credits to the leader. She checks the amount and then gives you a satisfied smile. Aty credits nod, the group vanishes into the life of the street.`
```

> 			`	You see movement in your peripheral vision just as you feel the credits pulled from your hand. One of the children off to your side darted forward before you could react and is now handing your credits to the leader. She checks the amount and then gives you a satisfied smile. Aty credits nod, the group vanishes into the life of the street.`

Reviewer: ```suggestion
			`	You see movement in your peripheral vision just as you feel the credits pulled from your hand. One of the children off to your side darted forward before you could react and is now handing your credits to the leader. She checks the amount and then gives you a satisfied smile. At her nod, the group vanishes into the life of the street.`
```
